chains/botanix/metadata.yaml

```diff
@@ -0,0 +1,29 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://api.routescan.io/v2/network/mainnet/evm/3637/etherscan/api
+    family: routescan
+    name: Botanix Explorer
+    url: https://botanixscan.io
+blocks:
+  confirmations: 1
+  estimateBlockTime: 6
+  reorgPeriod: 5
+chainId: 3637
+deployer:
+  name: Abacus Works
+  url: https://www.hyperlane.xyz
+displayName: Botanix
+domainId: 3637
+gasCurrencyCoinGeckoId: bitcoin
+name: botanix
+nativeToken:
+  decimals: 18
+  name: Bitcoin
+  symbol: BTC
+protocol: ethereum
+rpcUrls:
+  - http: https://botanixrpc.com
+technicalStack: other
+transactionOverrides:
+  maxFeePerGas: 1000007
```

Comment:
do you expect this to change much? as to my knowledge we've tended not to put tx overrides in the registry and have instead put them in infra, but happy either way if you think this won't cause problems. Also as we move further from infra tooling and to CLI this probably makes more sense

Reply:
will likely need to be tweaked later, still a new chain so some rough riding for users. but without this change, anyone will pay 1000x more than required. fun fact we spent more gas on 2txs using the default chain's gas price than the entire core deploy itself with the capped fee